commands/command_filter_process.go

```diff
@@ -89,11 +63,16 @@ func filterCommand(cmd *cobra.Command, args []string) {
 			err = clean(w, req.Payload, req.Header["pathname"])
 		case "smudge":
 			w = git.NewPktlineWriter(os.Stdout, smudgeFilterBufferCapacity)
-			err = filterSmudge(w, req.Payload, req.Header["pathname"], skip, filter)
+			err = smudge(w, req.Payload, req.Header["pathname"], skip, filter)
```

Comment:
As an added benefit, I got rid of the intermediate `filterSmudge` function, so now the legacy `filter.lfs.{clean,smudge}` and `filter.lfs.process` share the _exact same_ implementation! ✨